### DISCUSSION TITLE
Dictation and punctuate in spanish Nova-2

### DISCUSSION THREAD

**Discussion by rmasad** [deepgram/community#895] [General help] [answered]
I'm trying Deepgram as an alternative for a medical voice recognition system in Spanish. I'm interested in allowing users to dictate with spoken punctuation, which I read in Deepgram is activated with "dictation=True, punctuate=True". I try it and it doesn't work. Is this enabled in Spanish?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by rmasad** [comment]
The `request_id` is `fc529a65-8bde-4631-8f65-9bf17cd259e6`.

Here is the code I use:
```python
import os
from dotenv import load_dotenv
import logging
from deepgram.utils import verboselogs
from datetime import datetime
import httpx

from deepgram import (
    DeepgramClient,
    DeepgramClientOptions,
    PrerecordedOptions,
    FileSource,
)

load_dotenv()

AUDIO_FILE = "audios/1.wav"

DEEPGRAM_API_KEY = os.getenv("DEEPGRAM_API_KEY")

def main():
    try:
        # STEP 1 Create a Deepgram client using the API key in the environment variables
        config: DeepgramClientOptions = DeepgramClientOptions(
            verbose=verboselogs.SPAM,
        )
        deepgram: DeepgramClient = DeepgramClient(DEEPGRAM_API_KEY, config)
        # OR use defaults
        # deepgram: DeepgramClient = DeepgramClient()

        # STEP 2 Call the transcribe_file method on the rest class
        with open(AUDIO_FILE, "rb") as file:
            buffer_data = file.read()

        payload: FileSource = {
            "buffer": buffer_data,
        }

        options: PrerecordedOptions = PrerecordedOptions(
            model="nova-2",
            language="es-419",
            dictation=True,
            punctuate=True,
        )

        before = datetime.now()
        response = deepgram.listen.rest.v("1").transcribe_file(
            payload, options, timeout=httpx.Timeout(300.0, connect=10.0)
        )
        after = datetime.now()

        print(response.to_json(indent=4))
        print("")
        difference = after - before
        print(f"time: {difference.seconds}")

    except Exception as e:
        print(f"Exception: {e}")


if __name__ == "__main__":
    main()
```

**Comment by jpvajda** [comment]
@rmasad  Punctuation should work on Nova-2, but Dictation is an older feature of ours that might not. Let me check with a few people on the range of models Dictation currently supports.

**Comment by jpvajda** [comment, marked as answer]
@rmasad From my testing it looks like Dictation works with English only. I did update [our Docs](https://developers.deepgram.com/docs/dictation) on this feature to make that more clear.

**Reply by rmasad** [reply]
Thanks. Could it be available in Spanish at some point?

**Reply by jpvajda** [reply]
@rmasad  I can mention this to our product team for consideration!  To clarify Punctuation does has [multi-language support](https://developers.deepgram.com/docs/punctuation) currently. [Dictation](https://developers.deepgram.com/docs/dictation) is English only.
